share/man/man9/intro.9

```diff
@@ -448,9 +449,9 @@ API.
 .Pp
 Kernel sanitizers can perform additional compiler-assisted checks against
 memory use/access.
-These runtimes are capable of detecting difficult-to-identify classes of bugs,
+These runtimes are capable of detecting difficult-to-identify classes of bug,
```

Comment:
No, this just seems wrong.

Reply:
Reverted by [`0c4471a` (#813)](https://github.com/freebsd/freebsd-src/pull/813/commits/0c4471aaeecbc50260cd4310042441e838519ba6).